src/utils/removeUnnecessarySpaces.ts

```diff
@@ -0,0 +1,7 @@
+import { isString } from "lodash";
+
+const removeUnnecessarySpaces = (value?: string): string => {
+  return !isString(value) ? "" : value.replace(/\s+/g, ' ').trim()
```

Comment:
Not sure what the value lodash brings here.
```
return (value ?? '').replace(/\s+/g, ' ').trim()
```

Reply:
Here are the tests with all checks https://github.com/DeskproApps/lansweeper/pull/5/files#diff-ac204642de2401e350a9fa0c9fad82c6de55d48c60f01faf71690f2c6d6ea91c

Maybe it's redundant, but I'm like to double-checking

Reply:
Fixed. Removed unnecessary code